dialogue-core/src/main/java/com/palantir/dialogue/core/QueuedChannel.java

```diff
@@ -149,13 +167,15 @@ private boolean scheduleNextTask() {
         // There's a race where cancel may be invoked between this check and execution, but the scheduled
         // request will be quickly cancelled in that case.
         if (queuedResponse.isDone()) {
+            queueSizeEstimate.decrementAndGet();
             return true;
         }
         try (CloseableSpan ignored = components.span().childSpan("Dialogue-request-scheduled")) {
             Endpoint endpoint = components.endpoint();
```

Comment:
could we rename `components` to `headOfTheQueue` or something? components seems unnecessarily ambiguous

Reply:
👍